Directory.Build.props

```diff
@@ -89,6 +90,8 @@
     <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
     <!-- Used for launchSettings.json and runtime config files. -->
     <AppDesignerFolder>Properties</AppDesignerFolder>
+    <!-- Determine what architecture we are building on. -->
+    <BuildArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant())</BuildArchitecture>
```

Comment:
Does this dictates what architecture the product targets when building? Or can the target arch be overridden via an argument?

Reply:
This is the architecture the build process is running on, separate from the target architecture.